src/index.js

```diff
@@ -44,11 +46,21 @@ const component = () => {
   element.textContent = _.join(['Made', 'By SAIDI AZARIA'], ' ');
   return element;
 };
+/**
+ * this function load images and  give them
+ * some events
+ */
 const loadIconAndEvent = () => {
   const squares = document.querySelectorAll('.square');
```

Comment:
- [ ] Nice work done here with the square images but for this project, you are required to use the `input` tag to create the [checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox). You will need to implement the status updates (completed: true/false) feature using the `change` event handler in one file and import it as a module in your main file.

Here is a picture of these requirements

![Screenshot from 2021-09-30 10-51-43](https://user-images.githubusercontent.com/40503704/135430529-4c92138f-f9ca-4fc5-9497-5c4b483e2109.png)